### ISSUE TITLE
deactivate_user(username) return 401

### ISSUE THREAD

**Issue by alexanderreinelt** [pycontribs/jira#667]
**Describe the bug**
When using [deactivate_user(username)](https://jira.readthedocs.io/en/master/_modules/jira/client.html#JIRA.deactivate_user), an error message is returned telling me the following. 
```
text: User failed to authenticate
````
When trying any other operation (delete_user(username)), than deactivate_user the user with the same credentials it works perfectly. So the problem is not that I don't have valid credentials.

**To Reproduce**

Try to use deactivate_user(username) with basic authentication on a Jira Cloud Instance 

**Expected behavior**
user gets disabled and method is returning ```True```

**Stack Trace**

```
WARNING:root:Got recoverable error from POST https://domain.atlassian.net/admin/rest/um/1/user/deactivate?username=peter.pan, will retry [1/3] in 18.438257315360556s. Err: 401
WARNING:root:Got recoverable error from POST https://domain.atlassian.net/admin/rest/um/1/user/deactivate?username=peter.pan, will retry [2/3] in 25.891688874514557s. Err: 401
WARNING:root:Got recoverable error from POST https://domain.atlassian.net/admin/rest/um/1/user/deactivate?username=peter.pan, will retry [3/3] in 5.049205019874988s. Err: 401
Error Deactivating peter.pan: JiraError HTTP 401 url: https://domain.atlassian.net/admin/rest/um/1/user/deactivate?username=peter.pan
```

```
response headers = {'Server': 'Atlassian Proxy/0.1.305', 'Content-Type': 'text/plain;charset=UTF-8', 'Strict-Transport-Security': 'max-age=315360000; includeSubDomains; preload', 'Date': 'Fri, 03 Aug 2018 10:33:19 GMT', 'X-RateLimit-Limit': '800', 'X-RateLimit-Reset': '1533292409', 'X-Application-Context': 'application:prod-east:8080', 'IdPlat-Outstanding-Token': 'none', 'X-XSS-Protection': '1; mode=block', 'X-Content-Type-Options': 'nosniff', 'Connection': 'keep-alive', 'X-RateLimit-Remaining': '796', 'Content-Length': '27'}
```

**Version Information**
Python Interpreter: 3.7
jira-python: 2.0.0
OS: Mac OS High Sierra (10.13.6)

**Comment by xmatthias** [comment]
This seems to have been fixed in d8fc58d09e5090cbd354202539f6af27955570b1 - but no new Release has been made since then.
best install from source - then it will work (however without proxy, as there is still a bug in the implementation regarding that).

i'll create a PR to add the proxy support

**Comment by dobry-kod** [comment]
I cloned the master and made a link to / usr / local / lib / site-packages / jira but his not working.

WARNING:root:Got recoverable error from POST https://<*>.atlassian.net/rest/auth/1/session, will retry [1/3] in 8.19922155883183s. Err: 401
